oct/cli/provision/remote/tests/test_all_in_one.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function
+
+from mock import patch
+import oct.cli.provision.remote.all_in_one as all_in_one_module
+from oct.cli.provision.remote.all_in_one import OperatingSystem, Stage, provision_with_aws
+from oct.tests.unit.playbook_runner_test_case import PlaybookRunnerTestCase, \
+    TestCaseParameters, show_stack_trace, PlaybookRunCallSpecification
+
+if not show_stack_trace:
+    __unittest = True
+
+
+class ProvisionAWSTestCase(PlaybookRunnerTestCase):
+    def setUp(self):
+        PlaybookRunnerTestCase.setUp(self)
+        patches = [
+            patch.object(
+                target=all_in_one_module,
+                attribute='provision_with_aws',
+                new=lambda configuration, _, _, _, _: None,
+                # TODO: Need to patch private_key_path and keypair_name
```

Comment:
Hrm, this patches the methods of the class and there is no `aws_client_configuration` method (although an instance of Configuration has the field) so I end up hitting `AttributeError: <class 'oct.config.configuration.Configuration'> does not have the attribute 'aws_client_configuration'`. I will loop in @bbguimaraes tomorrow since he knows more about Python.